index.test.js

```diff
@@ -100,4 +99,27 @@ describe('babel-preset-zapier', () => {
       expect(transform(code)).toMatchSnapshot();
     });
   });
+
+  describe('when passing using the target option', () => {
+    const tetstConfig = options =>
+      require('./index')(
+        {
+          assertVersion: () => true,
+          cache: {
+            using: () => {},
+          },
+        },
+        options
+      );
+
+    it('produce the appropriate configuration for browser', () => {
```

Comment:
The test description reads kinda weird: 
`babel-preset-zapier when passing using the target option produce the appropriate configuration for node`.

Maybe we could get rid of the describe entirely and do `it('produces the correct config when target option is "browser"`)?

Reply:
I've used the already existing convention. I think it makes sense to regroup tests in a `describe` when they are all related to a given context/condition.

Usually, I use the word `given`: `babel-preset-zapier given the target option produces the appropriate configuration for node`. I will replace by that for now. We still can revisit this later anyway!